### PR TITLE
update urls to be in line with django 2.0 conventions

### PR DESCRIPTION
**Pull request by lpmi-13**
update to use path instead of url, which seems to be recommended with Django 2.x, though happy to discuss if that's not the way to go.

### CODE REVIEW

**Review by LilyFirefly — APPROVED** [review]

File: oxfordpython/urls.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
-from django.conf.urls import url
+from django.urls import path 
 from django.contrib import admin
 
 from .main import views as main_views
 
 urlpatterns = [
-    url(r'^admin/', admin.site.urls),
-    url(r'^$', main_views.HomeView.as_view(), name='home'),
+    path('admin', admin.site.urls),
```
Comment (craigloftus) [inline oxfordpython/urls.py:7]: This should be `admin/`; You end up with urls like `adminlogin` otherwise.

**Review by craigloftus — CHANGES_REQUESTED** [review]
The PR isn't based on the HEAD of the single_page branch, it hasn't caused a merge conflict in this instance but it did make testing it a bit confusing.

**Comment by lpmi-13** [comment]
Ah of course. Forgot to pull on single_pages after my own PR was merged.
Will sort!

**Comment by lpmi-13** [comment]
decided to close this, as I wanted to add some test cases and starting from a new branch seemed sensible for that fresh clean feeling.

**Pull request closed without merging**